web_notify/__manifest__.py

```diff
@@ -6,9 +6,10 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """ACSONE SA/NV,Odoo Community Association (OCA)""",
```

Comment:
Please revert this change. The author tag should just be standard quote, not multiline